src/components/FormInput.jsx

```diff
@@ -48,6 +48,21 @@ export const FormInput = styled.input`
       font-size: 1rem;
     }
   }
+
+  @media (max-width: 1100px) {
+    width: 55%;
+  }
+  @media (max-width: 900px) {
+    width: 55%;
+  }
+  @media (max-width: 500px) {
+    width: 55%;
+  }
+  @media (max-width: 400px) {
+    width: 55%;
+  }
```

Comment:
When you use percentage or relative size units, please add lower and upper bound, as a good practice, for example:

`min-width` and `max-width`

@Katsu08  please check good values to set here